src/test/java/org/systemexception/lunarlander/test/PhysicsTest.java

```diff
@@ -0,0 +1,29 @@
+package org.systemexception.lunarlander.test;
+
+import org.junit.Test;
+import org.systemexception.lunarlander.model.Physics;
+
+import java.util.HashMap;
+import java.util.List;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author leo
+ * @date 28/12/15 21:42
+ */
+public class PhysicsTest {
+
+	private Physics sut;
+
+	@Test
+	public void run_simulation_physics() {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/leo_4/lunar-lander/file/1677825522/issues/source?bid=3040650&fileBranchId=3040754#l20)